src/middlewares/tokenAuthentication.js

```diff
@@ -6,8 +6,9 @@ const verifyToken = (req, res, next) => {
     return res.sendStatus(403);
   }
   const token = authHeader.split(" ")[1];
-
-  const user = jwt.decode(token, process.env.ACCESS_TOKEN_SECRET);
+  if (!token) { return res.sendStatus(401) }
+  
+  const user = jwt.verify(token, process.env.ACCESS_TOKEN_SECRET, {expiresIn: "24h"}) 
```

Comment:
this function was created in util.jwt

Reply:
@Duncanwar  I replaced `jwt.decode` with `jwt.verify()` because `jwt.decode()` doesn't check if the token is expired!